homeassistant/components/tado/__init__.py

```diff
@@ -94,7 +99,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
     except requests.exceptions.Timeout as ex:
         raise ConfigEntryNotReady from ex
     except requests.exceptions.HTTPError as ex:
-        if ex.response.status_code > 400 and ex.response.status_code < 500:
+        if (
+            ex.response.status_code > HTTP_BAD_REQUEST
+            and ex.response.status_code < HTTP_INTERNAL_SERVER_ERROR
```

Comment:
This feels slightly icky to me, the existing numeric comparison is kind of clearer. Not a blocker either way as far as I'm concerned though, raising just in case you wish to reconsider.

Reply:
Completely agree, reverted my changes in this file.